test/geojson/test.ml

```diff
@@ -81,6 +81,30 @@ let test_multi_line () =
     t;
   Alcotest.(check ezjsonm) "same json" json json'
 
+let test_multi_point () =
+  let s = read_file "files/valid/multipoint.json" in
+  let json = Ezjsonm.value_from_string s in
+  let geo = Geojson.of_json json in
+  let coords =
+    match geo with Ok (Geometry (MultiPoint p)) -> p | _ -> assert false
+  in
+  let json' = Geojson.Geometry.MultiPoint.to_json coords in
+  let t =
+    Geojson.Geometry.(
+      Array.map (fun l -> [| Position.long l; Position.lat l |])
+      @@ MultiPoint.coordinates coords)
+  in
+
+  Alcotest.(check (array @@array (float 0.)))
```

Comment:
```suggestion
  Alcotest.(check (array array (float 0.)))
```
Did the `@@` raise a syntax error ?

Reply:
Hi @patricoferris Sir, no sir , not at all . It's working fine! without any syntax error.

Reply:
In fact if we remove `@@` _then that causes an error:_

![image](https://user-images.githubusercontent.com/65283880/160890821-7e6fe11e-3977-4590-b3ba-9f30482b7393.png)